docker/redpanda/entrypoint.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+if [[ ! -r /mnt/vectorized/redpanda.deb ]]; then
+  echo 'error: unable to read /mnt/vectorized/redpanda.deb'
+  exit 1
+fi
+
+dpkg --force-confold -i /mnt/vectorized/deb/redpanda.deb
```

Comment:
there is a typo, should be

`dpkg --force-confold -i /mnt/vectorized/redpanda.deb`

not

`dpkg --force-confold -i /mnt/vectorized/deb/redpanda.deb`

Reply:
ah, nice catch. i'll fix.